src/index.ts

```diff
@@ -18,10 +18,11 @@ export default function useHotjar(): IUseHotjar {
     (
       hotjarId: number,
       hotjarVersion: number,
+      hotjarDebug?: boolean,
```

Comment:
Could we pass it default to false?

`hotjarDebug = false,`

and then in the init below 

`hotjarInitScript(hotjarId, hotjarVersion, hotjarDebug);`